scripts/lib.sh

```diff
@@ -245,18 +287,21 @@ get_extensions_from_dynatrace() {
   cd ${CLUSTER_EXTENSIONS_TMPDIR} || exit
 
   EXTENSIONS_TO_DOWNLOAD="com.dynatrace.extension.google"
-  readarray -t EXTENSIONS_FROM_CLUSTER_ARRAY <<<"$( echo "${EXTENSIONS_FROM_CLUSTER}" | sed 's/ /\n/' | grep "$EXTENSIONS_TO_DOWNLOAD" )"
+  readarray -t EXTENSIONS_FROM_CLUSTER_ARRAY <<<"$(echo "${EXTENSIONS_FROM_CLUSTER}" | sed 's/ /\n/' | grep "$EXTENSIONS_TO_DOWNLOAD")"
   for i in "${!EXTENSIONS_FROM_CLUSTER_ARRAY[@]}"; do
     EXTENSION_NAME="$(cut -d':' -f1 <<<"${EXTENSIONS_FROM_CLUSTER_ARRAY[$i]}")"
     EXTENSION_VERSION="$(cut -d':' -f2 <<<"${EXTENSIONS_FROM_CLUSTER_ARRAY[$i]}")"
-    
+
     curl -k -s -X GET "${DYNATRACE_URL}/api/v2/extensions/${EXTENSION_NAME}/${EXTENSION_VERSION}" -H "Accept: application/octet-stream" -H "Authorization: Api-Token ${DYNATRACE_ACCESS_KEY}" -o "${EXTENSION_NAME}-${EXTENSION_VERSION}.zip"
     if [ -f "${EXTENSION_NAME}-${EXTENSION_VERSION}.zip" ] && [[ "$EXTENSION_NAME" =~ ^com.dynatrace.extension.(google.*)$ ]]; then
-      find ${EXTENSIONS_TMPDIR} -regex ".*${BASH_REMATCH[1]}.*" -exec rm -rf {} \;
-      mv "${EXTENSION_NAME}-${EXTENSION_VERSION}.zip" ${EXTENSIONS_TMPDIR}
+      check_gcp_config_in_extension "${EXTENSION_NAME}-${EXTENSION_VERSION}.zip"
+      if [ -f "${EXTENSION_NAME}-${EXTENSION_VERSION}.zip" ]; then
```

Comment:
`check_gcp_config_in_extension` call and if statement should be removed.
Lines 299 and 300 should stay.

Reply:
the line numeration is a little bit different, this is ok? https://github.com/dynatrace-oss/dynatrace-gcp-function/pull/214/commits/ee73793f2a1960f461bd15364a9ecba9fa23d7c4